src/style.css

```
@@ -0,0 +1,233 @@
+
+.imgLogIn{
+ 
+  width: 100%;
+  height: 30%;
+  left: 0px;
+  top: 0px;
+  border-radius: 92px;
+}
+
+.logoLogIn{
+
+width: 33%;
+height: 20%;
+left: 50%;
+top: 30%;
+  
+}
+
+
+#logIn{
+  background: linear-gradient(90deg, #D74177 -8.64%, #FFE98A 100%);
+  font-weight: bold;  
+  color: #FFFFFF;
+  width: 90%;
+  border: 0px;
+}
+
+#btngoogle{
+  background-image: url(https://image.flaticon.com/icons/png/512/60/60543.png);
+  background-position: center;
+  background-size: 43px;
+  height: 43px;
+  width: 43px;
+  border: 0px;
+}
+
+#btnface{
+  background-image: url(https://imageog.flaticon.com/icons/png/512/59/59439.png?size=1200x630f&pad=10,10,10,10&ext=png&bg=FFFFFFFF);
+  background-position: center;
+  background-size: 75px;
+  height: 40px;
+  width: 43px;
+  border: 0px;
+}
+
+#liga{
```

cuidado con el español 
```suggestion
#link{
```